### PR TITLE
chore: bump blocksuite

### PR DESCRIPTION
**Pull request by Flrande**

### CODE REVIEW

File: packages/frontend/hooks/src/use-block-suite-page-references.ts [hunk 1]
```diff
@@ -5,12 +5,9 @@ import { useBlockSuiteWorkspacePage } from './use-block-suite-workspace-page';
...
 const weakMap = new WeakMap<Page, Atom<string[]>>();
 function getPageReferences(page: Page): string[] {
-  // todo: is there a way to use page indexer to get all references?
-  return ['affine:paragraph', 'affine:list', 'affine:database']
-    .flatMap(f => page.getBlockByFlavour(f))
-    .flatMap(b => b.text?.toDelta())
-    .map(v => v?.attributes?.reference?.pageId)
-    .filter(Boolean);
+  return Object.values(
+    page?.workspace.indexer.backlink.linkIndexMap[page.id] ?? {}
```
Comment (EYHN) [inline packages/frontend/hooks/src/use-block-suite-page-references.ts]:
```suggestion
    page.workspace.indexer.backlink.linkIndexMap[page.id] ?? {}
```

File: packages/frontend/hooks/src/use-block-suite-page-preview.ts [hunk 2]
```diff
@@ -60,7 +60,7 @@ export function useBlockSuitePagePreview(page: Page | null): Atom<string> {
         page.slots.ready.on(() => {
           set(getPagePreviewText(page));
         }),
-        page.slots.yUpdated.on(() => {
+        page.slots.blockUpdated.on(() => {
```
Comment (EYHN) [inline packages/frontend/hooks/src/use-block-suite-page-preview.ts:63]: so block content update will trigger `blockUpdated` event?

Comment (Flrande) [reply packages/frontend/hooks/src/use-block-suite-page-preview.ts:63]: I think the preview doesn't need to be updated so promptly. If any changes in content will update the preview, I think it's easy to have performance issues.

**Review by joooye34 — APPROVED** [review]

**Comment by joooye34** [comment]
Do we have a changelog between those two versions?

**Pull request merged**